apis/status/v1beta1/constraintpodstatus_types.go

```diff
@@ -94,15 +94,12 @@ func NewConstraintStatusForPod(pod *corev1.Pod, constraint *unstructured.Unstruc
 	obj.SetNamespace(util.GetNamespace())
 	obj.Status.ID = pod.Name
 	obj.Status.Operations = operations.AssignedStringList()
-	cmVal, err := StatusLabelValueForConstraint(constraint)
-	if err != nil {
-		return nil, err
-	}
 	obj.SetLabels(map[string]string{
-		ConstraintMapLabel: cmVal,
-		PodLabel:           pod.Name,
+		ConstraintNameLabel: constraint.GetName(),
+		ConstraintKindLabel: constraint.GetKind(),
```

Comment:
where do we enforce less than 63 character limits for constraint template name and pod name for these labels?

Reply:
In my testing, kinds must be <= 63 characters long, and the template name must be the lowercase of the kind.

The pod name is chosen by us, so we have control there.